GMC_identification.ipynb

```diff
@@ -0,0 +1,1603 @@
+{
```

Comment:
<p>Ah -- now I see why 30. This could be made more explicit (i.e., write in "in this instance our beam is composed of 30 pixels)", or you could alter 30 to instead be something like omegaBeam/cube.pixelsize**2.0)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>